lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -193,11 +200,26 @@ def execute_statement(statement, parameters)
     rescue Sequel::DatabaseConnectionError, Sequel::DatabaseError => e
       @logger.warn("Exception when executing JDBC query", :exception => e)
     else
-      @sql_last_start = sql_last_start
+      @sql_last_value = sql_last_value
     end
     return success
   end
 
+  public
+  def get_column_value(row)
+    if !row[@tracking_column.to_sym]
```

Comment:
the column may exist, but be `nil`.

I would either warn on `row[@tracking_column.to_sym].nil?` or `row.has_key?(@tracking_column.to_sym)`,
and then the warning message would be unique to either. what do you think?